test/e2e/page-objects/pages/homepage.ts

```diff
@@ -34,6 +39,8 @@ class HomePage {
     css: '.transaction-status-label--failed',
   };
 
+  private readonly popoverBackround = '.popover-bg';
```

Comment:
minor typo
```suggestion
  private readonly popoverBackground = '.popover-bg';
```

Reply:
Nice catch, thanks !  Fixed in commit [506d3d9](https://github.com/MetaMask/metamask-extension/pull/28161/commits/506d3d9a419e88f5e9f7f0046553f03a9145bd27)